packages/color-block/src/components/list/ListItem.tsx

```diff
@@ -0,0 +1,140 @@
+                    onConfirm={(color) => {
+                        onUpdate({ ...color, alpha: (color.alpha && Math.round(color.alpha * 255)) || 255 });
+                    }}
+                >
+                    <div className="tw-h-full tw-mr-9 tw-bg-[url('https://cdn.frontify.com/img/transparent.png')] tw-bg-[length:10px_10px]">
+                        <div
+                            data-test-id="color-color-picker-flyout-trigger"
+                            className="tw-w-[120px] tw-h-full tw-min-h-[60px] tw-shadow-t-inner-line tw-transition-all group-hover:tw-shadow-t-inner-line-strong"
+                            style={{
+                                backgroundColor: getRgbaColorValue(color),
+                            }}
+                        />
+                    </div>
+                </ColorPickerFlyout>
+            ) : (
+                <Tooltip
+                    withArrow
+                    position={TooltipPosition.Right}
+                    hoverDelay={0}
+                    content={<TooltipContent colorValue={mappedFirstColorSpace.value ?? ''} status={status} />}
+                    triggerElement={
+                        <div className="tw-h-full tw-mr-9 tw-bg-[url('https://cdn.frontify.com/img/transparent.png')] tw-bg-[length:10px_10px]">
+                            <div
+                                data-test-id="color-tooltip-trigger"
+                                className="tw-w-[120px] tw-h-full tw-min-h-[60px] tw-cursor-pointer tw-shadow-t-inner-line tw-transition-all group-hover:tw-shadow-t-inner-line-strong"
+                                style={{
+                                    backgroundColor: getRgbaColorValue(color),
+                                }}
+                                onClick={() => copy(mappedFirstColorSpace.value ?? '')}
+                            />
+                        </div>
+                    }
+                />
+            )}
+
+            {isEditing && (
+                <div
+                    data-test-id="delete-button"
+                    className={joinClassNames([
+                        'tw-absolute tw-hidden tw-top-1/2 tw-right-3 tw-translate-y-[-50%]',
+                        isEditing && 'group-hover:tw-block',
+                    ])}
+                >
+                    <Button
+                        icon={<IconTrashBin size={IconSize.Size20} />}
+                        style={ButtonStyle.Secondary}
+                        onClick={() => onDelete(color.id)}
+                    />
+                </div>
+            )}
+
+            <ColorName
+                viewType={ColorBlockType.List}
+                initialColorName={color.name || ''}
+                isEditing={isEditing}
+                onBlur={onBlur}
+            />
+
+            <div className="tw-flex tw-flex-[0_0_calc(100%-306px)] tw-items-center tw-flex-wrap tw-grow tw-gap-y-2.5 tw-py-5">
```

Comment:
That `tw-flex-[0_0_calc(100%-306px)]` feels weird and it works fine without (same for other occurences)

Reply:
Without this, the layout breaks when all color spaces are selected. But I found a cleaner alternative.